clients/privacy-center/packages/fides-consent/src/lib/cookie.ts

```diff
@@ -25,8 +28,37 @@ const CODEC: Types.CookieCodecConfig<string, string> = {
   encodeValue: encodeURIComponent,
 };
 
+export const makeDefaults = ({
+  config,
+  context,
+}: {
+  config: ConsentConfig;
+  context: ConsentContext;
+}): CookieKeyConsent => {
+  const defaults: CookieKeyConsent = {};
+  config.options.forEach(({ cookieKeys, default: current }) => {
+    if (current === undefined) {
+      return;
+    }
+
+    const value = resolveConsentValue(current, context);
+
+    cookieKeys.forEach((cookieKey) => {
+      const previous = defaults[cookieKey];
+      if (previous === undefined) {
+        defaults[cookieKey] = value;
+        return;
+      }
+
+      defaults[cookieKey] = previous && value;
```

Comment:
I'm a little confused on what this value ends up being when setting `defaults[cookieKey]` like this. Does this simply return the boolean?

Reply:
It's possible* to configure two consent options (data uses) to target the same key, where one defaults to `true` while the other defaults to `false`. This ensures that the more restrictive default is used so that we don't opt the user into the consent that should be opted out.

\* Possible, but probably a bad idea because it's hard to reason about.